docs/discover-operator-presence.md

```diff
@@ -0,0 +1,23 @@
+# How do I discover the presence/availability of an Operator?
+
+## Discovering installed operators
+Checking the presence of custom installed operators in the cluster is simply a matter of checking all the 
+`ClusterServiceVersions` (CSV) that are available in the cluster. Every  operator installed by OLM has a corresponding CSV
+associated with it that that contains all the details of the operator. Running `kubectl get csvs -A`
+would return all CSVs across all namespaces and provide a high-level view of all custom installed operators. 
+
+If the ClusterServiceVersion fails to show up or does not reach the `Succeeded` phase, please check the [troubleshooting documentation](https://) to debug your installation.  
+
+## Finding available operators
+Operators are available from a variety of sources. The most straightforward option is to use the OperatorHub, an online catalog
```

Comment:
Would it be better to differentiate between "find operators that I can install from my cluster" and "find operators I can make available to install from my cluster" ? OperatorHub.io just provides you a list of stuff you could put on your cluster. The default install of OLM includes a catalogsource that aggregates these and makes them available on the cluster (https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/manifests/0.13.0/0000_50_olm_17-upstream-operators.catalogsource.yaml) but it's not required to install OLM. Maybe we should mention that as well as OperatorHub.io itself? 

OperatorHub.io is more like a reference page, you still need to add them to your cluster before you can install them.

Reply:
That's a good point @kevinrizza - I have updated the doc, hopefully I understood your meaning correctly. Let me know what you think 👍